src/wp-includes/global-styles-and-settings.php

```diff
@@ -225,6 +225,39 @@ function wp_get_global_stylesheet( $types = array() ) {
 	return $stylesheet;
 }
 
+/**
+ * Gets the global styles custom css from theme.json.
+ *
+ * @since 6.2.0
+ *
+ * @return string Stylesheet.
+ */
+function wp_get_global_styles_custom_css() {
+	// Ignore cache when `WP_DEBUG` is enabled, so it doesn't interfere with the theme developers workflow.
+	$can_use_cached = ! WP_DEBUG;
```

Comment:
See @hellofromtonya's feedback, at a minimum we should add the same `@todo` comment above here that is present in the other recent core commits where this check is introduced for a similar purpose.

Reply:
Looks like you already fixed that while I was reviewing 👍